### PR TITLE
Updating typings to support winston 3

### PR DESCRIPTION
**Pull request by C45tr0**
Not 100% sure this is all correct. It seems to work for my current setup.

### CODE REVIEW

**Comment by lazywithclass** [comment]
Thanks for this, I totally forgot to update them.

**Comment by yousefkhan** [comment]
When will this be merged?

File: package.json [hunk 1]
```diff
@@ -34,14 +34,14 @@
...
     "clarify": "^2.1.0",
     "coveralls": "^3.0.1",
     "istanbul": "^0.4.5",
     "mocha": "^5.2.0",
     "mockery": "^2.1.0",
     "npm-check-updates": "^2.14.2",
     "should": "^13.2.1",
-    "sinon": "^6.0.0"
+    "sinon": "^6.0.0",
+    "winston": "^3.0.0"
```
Comment (lazywithclass) [inline package.json]: Winston has been added as [peer dependency](https://nodejs.org/en/blog/npm/peer-dependencies/), [here](https://github.com/lazywithclass/winston-cloudwatch/blob/master/package.json#L23).

File: typescript/winston-cloudwatch.d.ts [hunk 2]
```diff
@@ -1,44 +1,37 @@
-import * as winston from 'winston';
+import * as winston from "winston";
+import * as Transport from "winston-transport";
 
-import { CloudWatch, CloudWatchLogs } from 'aws-sdk';
+import { CloudWatch, CloudWatchLogs } from "aws-sdk";
 
-interface CloudWatchIntegration {
```
Comment (lazywithclass) [inline typescript/winston-cloudwatch.d.ts]: You've removed lots of functions from here, I am not saying it's wrong, I'd like to have a code example on how you use this with TypeScript as I've never used this library with it.

**Comment by C45tr0** [comment]
The current release of 2.2.0 is working for me. Closing this.

**Pull request closed without merging**